LosAngelesDataProject/LADP
# map api implementation using leaflet.js library and maptiler api

### Screenshots
![image](
![Screenshot 2024-04-17 171445](https://github.com/LosAngelesDataProject/LADP/assets/149129171/82141ec0-e7b0-4fb9-ba89-3a7ab3e5056c)
)

### Changes
-Implemented Leaflet js Library - opensource
-Implemented Maptiler API for map-
-Created folder housing map component and osm-provider assets
-Added Map container CSS to App.css file

#### How to test
- To test at the BaseMap component to the App.jsx routes with path "/map", import BaseMap, then navigate to "/map" where map should render.
